ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1079,3 +1080,31 @@ literals:
 - zone_air_no2_probability_percentage_sensor
 - zone_air_no2_ozone_concentration_sensor
 
+# Added on 2021/10/22 for lighting control
+- lighting_switch_command:
+  - ON
+  - OFF
+- zone_lighting_switch_status:
+  - ON
+  - OFF
+- zone_lighting_switch_command:
+  - ON
+  - OFF
+- zone_brightness_percentage_command
+- lighting_switch_alarm:
```

Comment:
see comment above -- if this is needed, we should be more descriptive with what it does.

Reply:
upon checking the device here. it seems to be a general fault alarm; i will use "failed_alarm" instead of defining this. thanks sir.

Reply:
that sounds good. please delete this and update the types which use it.